subsys/suit/cache/src/suit_dfu_cache_rw.c

```diff
@@ -606,7 +617,7 @@ static suit_plat_err_t cache_0_update(uint8_t *address, size_t size)
 		}
 #endif /* CONFIG_SUIT_CACHE0_ERASE_ON_ENVELOPE_STORED */
 
-		return partition_initialize(&dfu_partitions_ext[0]);
+		return partition_initialize(&dfu_partitions_ext[0], false);
```

Comment:
I think instead of `false` it should be  `IS_ENABLED(CONFIG_SUIT_CACHE0_ERASE_ON_ENVELOPE_STORED)` and delete the part above

Reply:
Done